pages/_document/index.js

```diff
@@ -37,7 +38,6 @@ export default class MyDocument extends Document {
         </Head>
         <body>
           <Main />
-          <NextScript />
```

Comment:
Wondering if a static export has anything to do with it. On mobile so I can't investigate.